src/fdb5/api/FDB.h

```diff
@@ -120,6 +120,7 @@ class FDB {
     bool dirty() const;
 
     void registerCallback(ArchiveCallback callback);
+    void registerCallback(FlushCallback callback);
```

Comment:
registerFlushCallback()

Consider the case of a lambda or a nop function. Rename.